src/peft/utils/constants.py

```diff
@@ -207,6 +207,40 @@ def starcoder_model_postprocess_past_key_value(past_key_values):
     # "gemma": ["q_proj", "v_proj"],  # tested, does not work because of different shapes
 }
 
+TRANSFORMERS_MODELS_TO_FOURIERFT_TARGET_MODULES_MAPPING = {
```

Comment:
I assume that this is a 1:1 copy of `TRANSFORMERS_MODELS_TO_LORA_TARGET_MODULES_MAPPING`. In that case, you could also do:

`TRANSFORMERS_MODELS_TO_FOURIERFT_TARGET_MODULES_MAPPING = TRANSFORMERS_MODELS_TO_LORA_TARGET_MODULES_MAPPING.copy()`

However, note that not all LoRA models might work with FourierFT as LoRA supports more different layer types. E.g. did you test if FourierFT works with gpt2, which uses `Conv1D` layers? I think the others should be okay, but I don't know each and every architecture.

Reply:
Bump